cmake/dependencies/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+#####################
+##  Antares solver  ##
+#####################
+if(BUILD_antares_solver)
+
+set(REPOSITORY "https://github.com/AntaresSimulatorTeam/Antares_Simulator.git")
+set(TAG "develop")
+set(CMAKE_ARGS "-DBUILD_UI=OFF -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}")
+
+if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+  set(ANTARES_BUILD_TYPE "debug")
+else ()
+  
+  set(ANTARES_BUILD_TYPE "release")
+endif ()
+
+build_git_dependency(
+    NAME
+      antares-solver
+    REPOSITORY
+      ${REPOSITORY}
+    TAG
+      ${TAG}
+    CMAKE_ARGS
+      "${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=${ANTARES_BUILD_TYPE}"
+    SOURCE_SUBDIR
+      "src"
+  )
+  
+endif()
+
+#####################
+##  ORTOOLS  ##
+#####################
+if(BUILD_ortools)
+
+set(REPOSITORY "https://github.com/AntaresSimulatorTeam/or-tools.git")
+set(TAG "rte_dev_sirius")
+set(CMAKE_ARGS "-DUSE_SIRIUS=OFF")
+
+if (USE_ORTOOLS_STABLE)
+	set(REPOSITORY "https://github.com/google/or-tools.git")
+	set(TAG "stable")
+	set(CMAKE_ARGS "")
```

Comment:
This line does nothing

Reply:
Why ? We change CMAKE_ARGS to an empty string no ?